### ISSUE TITLE
Recently Played is Considered Ascending by Default

### ISSUE THREAD

**Issue by AskaLangly** [probablyraging/steam-game-idler#27]
### Describe the issue

The title says enough. Recently Played is apparently set to ascending and not descending. The first game that popped up for me was TF2, but I haven't played that since 2011-ish. Might I ask to have the ability to choose between ascended and descended? More so, change the category's name to "Last Played"?

Reproduce: Simply change the game sort to Recently Played.
Expected: Most recently played games would be listed first.

[log.txt](https://github.com/user-attachments/files/16920697/log.txt)

### Which branch of SGI?

Base

### What version of SGI are you using?

Latest version

### Additional Information

- [X] I have included steps to reproduce the issue
- [ ] I have included screenshots/videos
- [X] I have included a log.txt file
- [X] I have checked for similar issues before reporting

**Comment by probablyraging** [comment]
Thanks for reaching out.

The `recently played` sorting feature by default should sort your games in descending order, that being from most recently played to least recently played _(based on timestamp data returned by Valve's API)_.

Double check to make sure `my profile` and `game details` are both set to `public` in your Steam profile settings, and uncheck the `Always keep my total playtime private even if users can see my game details.` option, as I believe these settings prevents SGI from accessing the required data. It can take 10+ minutes for these changes to take affect.

Once done, you can refresh your games list by clicking the refresh button next to the pagination or by logging in and out of SGI.
![image](https://github.com/user-attachments/assets/25084274-c57b-4d01-8f46-f1b012d9f5df)

Thanks

**Comment by probablyraging** [comment]
Just seen your reply on Reddit regarding a portable version. There was a portable version available prior to [v1.3.3](https://github.com/probablyraging/steam-game-idler/releases/tag/1.3.3) but I decided to no longer include it in releases as I introduced auto-updates that would cause issue with the portable version, and to avoid that I would have to maintain a completely seperate branch just for portable and I didn't feel like it was worth the hassle.

Maybe in the future, once SGI is a bit more mature and I'm not actively adding/tweaking features for the base version, I will consider bundling the portable in releases again.

**Comment by probablyraging** [comment]
Sorry, I've actually found what is causing this issue, and am working on a fix now. I was sorting games by recently played by using the `rtime_last_played` property returned by the API when fetching games lists, I've just found out that this property is only present when fetching the games list for the account tied to the API key making the request only.

So instead I will use the [`IPlayerService#GetRecentlyPlayedGames`](https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames) endpoint. However, this endpoint doesn't return any sort of timestamp of when the game was played, so there will be no way to sort by ascending/descending. Similar to when you view your recently played games on Steam `https://steamcommunity.com/profiles/<steamId>/games/?tab=recent`

Will update this issue when the fix is live.
